pgml-dashboard/src/components/dropdown/template.html

```diff
@@ -0,0 +1,22 @@
+<div class="dropdown d-flex show">
+  <button class="horizontal-hide btn btn-dropdown dropdown-toggle expanded <% if collapsable { %> leftnav-collapse-affect <% } %>" href="#" role="button" data-bs-toggle="dropdown" data-bs-offset="0, 10" aria-expanded="false">
+    <span class="btn-dropdown-text"><%= value %></span>
+    <span class="material-symbols-outlined">
+      expand_more
+    </span>
+  </button>
+
+  <div class="menu-item horizontal-hide dropdown-toggle collapsed <% if collapsable { %> leftnav-collapse-affect <% } %>" href="#" role="button" data-bs-toggle="dropdown" data-bs-offset="68, -44" aria-expanded="false">
```

Comment:
We might want the ability to control this data-bs-offset.  It controls where the dropdown menu appears relative to its default position.  Also, I can see needing to position the dropdown list left center or right very useful.